### ISSUE TITLE
bin/admin/change-versions should update the main README file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3544]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3544|
|Type|defect|
|Created|2008-11-17 01:26:46Z|
|Branch|https://github.com/twisted/twisted/tree/change-main-readme-version-3544|

It only updates README and _version.py files from subprojects, but not the toplevel README.

<details><summary>Searchable metadata</summary>

```
trac-id__3544 3544
type__defect defect
reporter__radix radix
priority__normal normal
milestone__totally_automated_release_infrastructure totally automated release infrastructure
branch__branches_change_main_readme_version_3544 branches/change-main-readme-version-3544
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1226885206000000 1226885206000000
changetime__1227018982000000 1227018982000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [25389]) Branching to 'change-main-readme-version-3544'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner|
|-|-|

Straightforward change to `changeAllProjectVersions`.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @exarkun|
|-|-|

Nice, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [25414]) Merge change-main-readme-version-3544

Author: exarkun
Reviewer: therve
Fixes: [#3544](https://github.com/twisted/twisted/issues/3544)

Change `changeAllProjectVersions`, the main function used to implement the `change-versions`
script, so that it also changes the version information in the top-level README file.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
